### DISCUSSION TITLE
freeRouting 1.4.5-1 more aggressive in optimisation than 1.4.4, causing clearance issues with Kicad

### DISCUSSION THREAD

**Discussion by retiredfeline** [freerouting/freerouting#79] [Q&A]
I don't know if I should raise an issue so I'm asking first.

I have a design that is routed fine by 1.4.4 and has no DRC violations after importing the SES into Kicad 5.1.12. Today I downloaded 1.4.5-1 and tried it on the same design. It ran faster but the result had a few DRC track-pad violations when the SES was imported into pcbnew. I think in all the cases they involved a diagonal track passing too close to a pad. I noticed that 1.4.5-1 causes the tracks to hug the pads closer in pursuit of reducing total track length.

Are there some clearance rules I should be changing to prevent this, either in pcbnew or freeRouting?

I can raise this as an issue if this warranted. The design is mine and there are no copyright issues. My platform is Linux, but that shouldn't matter.

Thanks.

**Comment by andrasfuchs** [comment]
My suspicion is that this is a bug in freerouing. This is probably the same issue that the guys were discussing in #69.

It's very unlikely that I will have time to fix this anytime soon, but you can switch back to the single-thread route optimizer with the `-mt 1` command line argument. 
Could you give it a try and check if the problem still exists with that option?

**Comment by retiredfeline** [comment]
Yes going single-threaded seems to have prevented the generation of the violations.

Thanks for the work on freeRouting, I updated because of the log4j bug.
